Dockerfile.dev

```diff
@@ -0,0 +1,28 @@
+# Stage 1: Install Dependencies
+FROM node:lts AS dependencies
+
+WORKDIR /usr/src/app
+
+COPY package*.json .
+
+RUN npm install
+
+# Stage 2: Build
+FROM node:lts AS builder
+
+WORKDIR /usr/src/app
+
+COPY --from=dependencies /usr/src/app/node_modules ./node_modules
+
+COPY . .
+
+# Stage 3: Final image
+FROM node:alpine
```

Comment:
You can't have node:alpine as final image for “Development” Dockerfile as it will cause problems in dev environment